upng.js

```diff
@@ -0,0 +1,1046 @@
+
```

Comment:
Can't you just add a dependency for [upng-js](https://www.npmjs.com/package/upng-js) instead?

Reply:
Nope, it has no properly exported package anywhere, just clones from random persons :(

Thus it is safer to rely on commit export from repo.